style.css

```
@@ -0,0 +1,588 @@
+}
+.learn-more {
+    color: #540CFA;
+}
+.customized-item-machine-learning .customized-item-icon {
+    background: linear-gradient(135deg, #F2C314,#FC3832);
+}
+.customized-item-automated-tracking .customized-item-icon {
+    background: linear-gradient(135deg, #9C09DB,#1C0FD7);
+}
+.customized-item-smart-notifications .customized-item-icon {
+    background: linear-gradient(135deg, #32FBFC,#3214F2);
+}
+.movie-slides {
+    display: flex;
+    padding: 140px 0;
+}
+
+/* interface section */
+.interface-intro h2 {
+    font-size: 46px;
+    font-weight: 400;
+    max-width: 460px;
+    line-height: 1;
+}
+.interface-intro p {
+    max-width: 460px;
+    margin-bottom: 57px;
+}
+.interface-intro-btn {
+    display: flex;
+    align-items: center;
+    margin-bottom: 20px;
+    padding: 17px;
+    max-width: 340px;
+    opacity: 0.5;
+    transition: all .2s ease-in-out;
+    border-radius: 4px;
+    font-size: 16px;
+}
+
+.interface-intro-btn svg {
+    margin-right: 20px;
+    width: 20px;
+    height: 20px;
+}
+.interface-intro-btn:hover {
+    opacity: 1;
+    box-shadow: rgba(0, 0, 0, 0.1) 0 1px 3px 0, rgba(0, 0, 0, 0.05) 0 1px 2px 0;
+}
+.interface-intro {
+    float: left;
+    width: 42%;
+    margin-right:9%
+}
+.interface-img {
+    float: left;
+    width: 49%;
+    position: relative;
+}
+.interface-img-iphone {
+    position: relative;
+    z-index: 2;
+    left: 0;
+    margin-top: 100px;
+}
+.interface-img-web {
+    position: absolute;
+    top: -48px;
+    left: 88px;
+    z-index: 1;
+}
+.interface-section {
+    overflow: hidden;
+    padding-bottom: 100px;
+}
+
+/* community */
+.community-section {
+    color: white;
+    background-image: url(testimonials.png);
+    background-color: black;
+    background-repeat: no-repeat;
+    background-position-x: 725px;
+}
+.community-intro {
+    padding-top: 200px;
+    padding-bottom: 200px;
+    max-width: 460px;
+}
+.community-intro h2 {
+    font-size: 46px;
+    font-weight: 400;
+    line-height: 1.08;
+}
+/* get started */
+.get-started-section {
+    padding: 125px 0;
+}
+.get-started-section h2 {
+    font-size: 60px;
+    font-weight: 400;
+    max-width: 606px;
+    margin: 0 auto;
+    margin-bottom: 20px;
+    text-align: center;
+}
+.get-started-section p {
+    max-width: 520px;
+    margin: 0 auto;
+    text-align: center;
+    margin-bottom: 50px;
+}
+.get-started-section .email-wrapper {
+    margin: 0 auto;
+}
+
+/* footer */
+
+.social {
+    max-width: 130px;
+    margin: 0 auto;
+}
+.social a {
+    margin: 10px;
+    opacity: 0.2;
+    transition: all 0.2s ease-in-out;
+}
+.social a:hover {
+    opacity: 0.5;
+}
+.footer {
+    padding: 40px;
+}
+
+/* for phone */
+@media screen and (max-width: 785px) {
+    /* intro-section */
+    .navbar-menu {
+        position: fixed;
+        right: 0;
+        top: 50px;
+        height: 50%;
+        background: white;
+        width: 260px;
+        flex-direction: column;
+        transition: all .3s ease-in-out;
```

This should probably be changed to:

```suggestion
        transition: transform .3s ease-in-out;
```

We don't want to animate the padding and line height changes to the mobile menu.